app/js/components/mock/serviceCreateForm.html

```diff
@@ -0,0 +1,74 @@
+<form name="dashboard_bundle_service_type" method="post" novalidate="novalidate">
```

Comment:
The naming of the mock files should in my opinion also be in snake case. Also goes for the 'serviceEdit' counterpart.

Reply:
The naming of fixture files doesn't bother me that much, but I've changed it as suggested.